osconfig_tests/junit/junit.go

```diff
@@ -0,0 +1,137 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package junit
```

Comment:
It's really weird to see a Go file called `junit`. How about adding a comment about what this does and why we need it?

Reply:
Lol was supposed to have a package comment, not sure why lint didn't pick that up